pkg/openstack/loadbalancer.go

```diff
@@ -687,6 +688,7 @@ func getBoolFromServiceAnnotation(service *corev1.Service, annotationKey string,
 		case "false":
 			returnValue = false
 		default:
+			klog.Error("Found a non-boolean Service Annotation: %v = %v (falling back to default setting: %v)", annotationKey, annotationValue, defaultSetting)
```

Comment:
Good! Now return `defaultSetting` after this line.

Reply:
> Good! Now return `defaultSetting` after this line.

Done! Is it  cool now? @pierreprinetti

Reply:
Perfect! Thanks.